### PR TITLE
Add rpm-no-comps fixture

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
@ggainey here is my attempt at this no-comps.

**Review by ggainey — CHANGES_REQUESTED** [review]
Needs to have a separate [noissue] line in the commit-message as well.

File: Makefile [hunk 1]
```diff
@@ -523,6 +526,10 @@ fixtures/rpm-zstd-metadata: fixtures
 	rpm/gen-fixtures.sh $@ rpm/assets
 	createrepo_c --general-compress-type=zstd --update $@
 
+fixtures/rpm-no-comps: fixtures
```
[the remark this answers is not in the document]
Comment (chris1984) [reply Makefile:529]: That makes sense, the part about me breaking the repo with removing the `comps.xml` and the `repomd.xml` still pointing to it. The only 2 repos I found that didn't have a `comps.xml` was `assets-modularity` and `assets-modular` Would manually creating an assets directory from `assets` then updating the `repomd.xml` to not have any reference to `comps.xml` then have the make file use that directory or is there a better way forward?

[the remark this answers is not in the document]
Comment (ggainey) [reply Makefile:529]: The general pattern is, in the build-stanza in the Makefile for your fixture, you `cp -r` from assets into your chosen dir-name, remove comps.xml, and then gen-fixture the resulting directory.

**Comment by chris1984** [comment]
@ggainey I think I got it figured out, looks right locally based on your feedback. Updated my commit message to have `[noissue]` inside it as well on a new line. Let me know how this looks?

**Review by ggainey — APPROVED** [review]
Looks good!

**Pull request merged**